{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,157 +1,58 @@
-# {{ cookiecutter.project_name }}
-
-## Overview
+# `{{ cookiecutter.project_name }}`
```

Comment:
In the older version, we had a project structure diagram like this one https://github.com/drivendata/cookiecutter-data-science#the-resulting-directory-structure
which I think was really helpful to hep the user gain an understanding of the structure.

Not for this PR, but I'll add a Trello card as I think it'd be good to have it back in.

Reply:
Agreed - although I'd prefer if we don't put the diagram in the root `README.md` file, and instead link to it! Marking this as **Resolved**.